sohojogi/flutter_native_splash.yaml

```
@@ -0,0 +1,154 @@
+  # If the image_dark parameter is specified, color_dark or background_image_dark must be specified.
+  # color_dark and background_image_dark cannot both be set.
+  #color_dark: "#042a49"
+  #background_image_dark: "assets/dark-background.png"
+  #image_dark: assets/splash-invert.png
+  #branding_dark: assets/dart_dark.png
+
+  # From Android 12 onwards, the splash screen is handled differently than in previous versions.
+  # Please visit https://developer.android.com/guide/topics/ui/splash-screen
+  # Following are specific parameters for Android 12+.
+  android_12:
+  # The image parameter sets the splash screen icon image.  If this parameter is not specified,
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)